common/src/main/java/me/shedaniel/architectury/networking/simple/PacketID.java

```diff
@@ -0,0 +1,28 @@
+package me.shedaniel.architectury.networking.simple;
+
+import me.shedaniel.architectury.networking.NetworkManager;
+import net.minecraft.resources.ResourceLocation;
+
+public final class PacketID {
+    private final SimpleNetworkManager manager;
+    private final ResourceLocation id;
+    private final NetworkManager.Side side;
+    
+    PacketID(SimpleNetworkManager h, ResourceLocation i, NetworkManager.Side s) {
+        manager = h;
```

Comment:
should a packet id be coupled to a manager?

Reply:
I don't see a reason not to be, in case in future it's required. It's already using the manager to create it, might aswell just pass it onto id